descqa/CorrelationsTwoPoint.py

```diff
@@ -492,7 +498,8 @@ def run_on_single_catalog(self, catalog_instance, catalog_name, output_dir):
                 catalog_data, sample_conditions)
 
             with open(os.path.join(output_dir, 'galaxy_count.dat'), 'a') as f:
-                f.write('{} {}\n'.format(sample_name, len(tmp_catalog_data['ra'])))
+                f.write('{} {} {:.1f} sq. deg.\n'.format(sample_name, len(tmp_catalog_data['ra']),
+                                                          self.check_footprint(tmp_catalog_data))) #print area
```

Comment:
The footprint is actually calculated on the full set, not the tmp_catalog_data subsamples, so this output is a little misleading

Reply:
Changed message to read "Total... area"